src/utils/helpers.test.js

```diff
@@ -56,3 +69,123 @@ describe('getUKFormattedEuros', () => {
     expect(result).toBe('€123.45');
   });
 });
+
+describe('getCurrentMonth', () => {
+  afterEach(() => {
+    global.Date.now = realDateNow;
+  });
+
+  test('if the current date is 2022-06-01, it should return "June 2022"', () => {
+    const date = '2022-06-01';
+    global.Date.now = getDateNowStub(date);
+
+    const expected = 'June 2022';
+    const actual = getCurrentMonth();
+
+    expect(actual).toBe(expected);
+  });
+  test('if the current date is 2024-02-29, it should return "February 2024"', () => {
+    const date = '2024-02-29';
+    global.Date.now = getDateNowStub(date);
+
+    const expected = 'February 2024';
+    const actual = getCurrentMonth();
+
+    expect(actual).toBe(expected);
```

Comment:
would it be possible to write something like?

```js
    expect(getCurrentMonth()).toBe('February 2024');
```

Reply:
and if this makes sense, does it make sense to group tests together into one? (e.g. if we find that all tests have to do with formatting the correct month, we might as well put them in one test). I know this goes against the advice to have atomic tests. please try out what works best for you and what is more readable.